src/Modules/Workspaces/Commands.Workspaces/AddPowerBIWorkspaceUser.cs

```diff
@@ -28,10 +26,14 @@ public class AddPowerBIWorkspaceMember : PowerBICmdlet, IUserScope
         public PowerBIUserScope Scope { get; set; } = PowerBIUserScope.Individual;
 
         [Parameter(Mandatory = true)]
-        public Guid GroupId { get; set; }
+        [Alias("GroupId", "WorkspaceId")]
+        public Guid Id { get; set; }
 
         [Parameter(Mandatory = true)]
-        public GroupUserAccessRight UserDetails { get; set; }
+        public string UserEmailAddress { get; set; }
+
+        [Parameter(Mandatory = true)]
+        public string UserAccessRight { get; set; }
```

Comment:
It might be better to use the UserAccessRight enum to enforce what values can be passed in.

Reply:
Agree plus you get tab completion in PS

Reply:
This requires the groupUserAccessRightProperty which is a string value.